secboot/secboot_tpm.go

```diff
@@ -936,6 +936,9 @@ func mockableReadKeyFileImpl(keyFile string, keyLoader *mockableKeyLoader, hintE
 
 var mockableReadKeyFile = mockableReadKeyFileImpl
 
+// GetPCRHandle returns the handle used by a key.  The key will be
+// search on as token on node in the keySlot. If keySlot has no
```

Comment:
s/search on/searched/ ?

Reply:
Maybe "searched on the token of the keySlot from the node". I will change that.